assets/sass/local.scss

```diff
@@ -1,3 +1,7 @@
+a:visited {
+  color: $govuk-link-colour!important;
```

Comment:
Don't use `important`, add a new class to where you want to do this. That is going to override every anchor visited link everywhere. https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
changed to class implementation, thanks Paul!